src/request.ts

```diff
@@ -9,7 +9,7 @@ import { ParamList } from './UrlSearchParams';
 declare var require: any;
 
 export class FilterRegistry extends Registry<RequestFilter> {
-	register(test: string | RegExp | RequestFilterTest, value: RequestFilter, first?: boolean): Handle {
+	register(test: string | RegExp | RequestFilterTest | null, value: RequestFilter, first?: boolean): Handle {
```

Comment:
This is due to the `Registry` class specifying `null` as the test here https://github.com/dojo/core/pull/178/files#diff-b0f17e2a57f905a1a051ae1f86ebc3cdR58 because it is set to `null` here https://github.com/dojo/core/pull/178/files#diff-b0f17e2a57f905a1a051ae1f86ebc3cdR76